src/proxy.c

```diff
@@ -47,6 +47,7 @@ struct watcher_proxy_data
 	 * otherwise it WILL wait
 	 */
 	int         *oob_fd;
+	gboolean    oob_fd_found;
```

Comment:
you could remove this field and create a local variable in cc_proxy_msg_read since this variable is only used by this function